lib/config.js

```diff
@@ -69,26 +67,57 @@ const Config = function () {
   this.mac_signing_output_prefix = 'signing'
   this.notary_user = getNPMConfig(['notary_user']) || ''
   this.notary_password = getNPMConfig(['notary_password']) || ''
-  this.channel = ''
+  this.channel = 'development'
   this.sccache = getNPMConfig(['sccache'])
   this.braveReferralsApiKey = getNPMConfig(['brave_referrals_api_key']) || ''
   this.ignore_compile_failure = false
   this.enable_hangout_services_extension = true
   this.widevineVersion = getNPMConfig(['widevine', 'version'])
-  this.brave_enable_cdm_host_verification = false
   this.sign_widevine_cert = process.env.SIGN_WIDEVINE_CERT || ''
   this.sign_widevine_key = process.env.SIGN_WIDEVINE_KEY || ''
   this.sign_widevine_passwd = process.env.SIGN_WIDEVINE_PASSPHRASE || ''
   this.signature_generator = path.join(this.srcDir, 'third_party', 'widevine', 'scripts', 'signature_generator.py') || ''
   this.extraGnArgs = {}
   this.extraNinjaOpts = []
   this.braveSafetyNetApiKey = getNPMConfig(['brave_safetynet_api_key']) || ''
-  this.braveAndroidKeystorePath = getNPMConfig(['brave_android_keystore_path']) || ''
+  // Should not be empty as causes build error in gn 'You can't use empty strings as directories'
+  this.braveAndroidKeystorePath = getNPMConfig(['brave_android_keystore_path']) || '.'
```

Comment:
Only `brave_android_keystore_path` should not be empty. `braveAndroidKeystorePath`should be empty in this case. Otherwise it will try to sign apk.

Reply:
Or we should change this logic https://github.com/brave/brave-browser/blob/93a6b2c4c06e9ad9367781e6e23f02a871a9e565/lib/config.js#L267

Reply:
I think that needs to change because it doesn't make sense to me to set the default to two different things in two different places